bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -194,53 +288,35 @@ impl ClusterConfig {
     }
 
     /// Create a cluster with the given config.
+    #[allow(clippy::let_and_return)]
     pub fn create_cluster(&self) -> ProviderResult<()> {
-        let status = match self {
+        let cluster_config_path = match self {
             Self::Args {
                 cluster_name,
                 region,
                 version,
-                zones,
+                zones: _,
```

Comment:
I don't follow the change to understand why you needed to change this? Are we no longer handling zones? Won't this break code where we have to be very clear which zone we are launching in when there is capacity only in a certain zone?